test/retry.js

```diff
@@ -54,7 +54,7 @@ test.before('setup', async () => {
 });
 
 test('works on timeout error', async t => {
-	t.is((await got(`${s.url}/knock-twice`, {timeout: {connect: 100, socket: 100}})).body, 'who`s there?');
+	t.is((await got(`${s.url}/knock-twice`, {timeout: {connect: 500, socket: 100}})).body, 'who`s there?');
```

Comment:
`500` and `100` should be constants at the top (with descriptive names) and used here and elsewhere. Makes it much clearer and easier to change.

Reply:
Indeed.